run_inference.py

```diff
@@ -28,4 +62,6 @@ def main(input_file: str, model_path: str = typer.Option("khaimaitien/leetcode_s
 
 
 if __name__ == "__main__":
-    typer.run(main)
+    # typer.run(main)
+    reader = crawl_desc_leetcode_url("https://aaa.com/problems/substring-with-concatenation-of-all-words/")
```

Comment:
Can you give the real link? and allow user to pass real-link in command line?
give 2 options: read from file and read from url
--url https://leetcode.com/problems/two-sum/
or 
--file test_cases/problem1.txt

python run_inference.py --url https://leetcode.com/problems/two-sum/

Reply:
Also please update the readme

Reply:
@khaimt Thanks, I made changes, help me merge request!